pwndbg/color/context.py

```diff
@@ -1,107 +1,43 @@
-import pwndbg.color.theme as theme
-from pwndbg.color import generateColorFunction
+from pwndbg.color import ColorConfig
+from pwndbg.color import ColorParamSpec
 from pwndbg.gdblib import config
 
-config_prefix_color = theme.add_color_param(
-    "code-prefix-color", "none", "color for 'context code' command (prefix marker)"
+# TODO: Should probably change - to .
```

Comment:
Before making this change we should decide on exactly how we want the namespacing to look. Should a config option appear as `context-code-prefix-color`, `context.code-prefix-color`, or `context.code.prefix-color`? I prefer the last option, but I'm also OK with the second option.

An alternative here is I can disable the warning (while keeping the forwarding to the new parameters), and then we can just figure out how we want to do this later.

Reply:
@disconnect3d Ping on this question.

Reply:
Another option is: `context code prefix-color` but I think keeping it without spaces may make more sense.

Anyway, `context.code.prefix-color` or `context-code-prefix-color` i think